cmd/api/README.md

```diff
@@ -2,42 +2,36 @@
-|--------------------------|-----------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
-| DB_USER                  | Postgres Database user                                                                                                            | user                                                                                         |
-| DB_PASSWORD              | Postgres Database Password                                                                                                        | hunter2                                                                                      |
-| DB_HOST                  | Postgres Database host                                                                                                            | /cloudsql/my-project:us-east1:tekton-results                                                 |
-| DB_NAME                  | Postgres Database name                                                                                                            | tekton_results                                                                               |
-| DB_SSLMODE               | Database SSL mode                                                                                                                 | verify-full                                                                                  |
-| DB_SSLROOTCERT           | Path to CA cert used to validate Database cert                                                                                    | /etc/tls/db/ca.crt                                                                           |
-| DB_ENABLE_AUTO_MIGRATION | Auto-migrate the database on startup (create/update schemas). For further details, refer to <https://gorm.io/docs/migration.html> | true (default)                                                                               |
-| PROFILING                | Enable profiling server                                                                                                           | false  (default)                                                                             |
-| PROFILING_PORT           | Profiling Server Port                                                                                                             | 6060  (default)                                                                              |
-| DB_MAX_IDLE_CONNECTIONS  | The number of idle database connections to keep open                                                                              | 2 (default for golang, but specific database drivers may have settings for this too)         |
```

Comment:
You deleted all my tuning options here in the README @sayan-biswas .... we need to minially document them, even if they are not specified in the config/base/env/config file

Reply:
Ohh.. Not sure why how this was missed. I'll rebase again to include the changes.